### DISCUSSION TITLE
force mobile version

### DISCUSSION THREAD

**Discussion by rejetto** [jellyfin/jellyfin-web#4407] [General]
i'm using the web client after creating a "shortcut" with chrome, and using it as it was an app, but i like the mobile interface more.

Is there a way (url parameter) to force loading of the mobile version while on a desktop browser?
thanks

**Comment by rejetto** [comment]
i've found that it's responsive, so i can just resize the window.
sorry to bother you :)

**Comment by thornbill** [comment]
You can change the display mode by going to the user menu > display > set the display mode to "mobile".
